libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -222,6 +225,9 @@ class SRV_Channels {
     // set output value for a function channel as a pwm value
     static void set_output_pwm(SRV_Channel::Aux_servo_function_t function, uint16_t value);
 
+    // set output value for a function channel as 0 pwm
+    static void set_output_pwm_zero(SRV_Channel::Aux_servo_function_t function);
```

Comment:
I think that ```set_output_limit``` would actually be preferable, ```set_output_pwm_zero``` seems like it might do different things then just setting the output to 0. (It's also more longer :) )